basic/src/test/java/org/jboss/hal/testsuite/test/configuration/elytron/ElytronOperations.java

```diff
@@ -13,7 +13,6 @@
 
 /**
  * Encapsulates Creaper operations in Elytron subsystem
- *
```

Comment:
Please don't add or remove empty lines unless it is absolutely necessary. It makes history less readable.